### DISCUSSION TITLE
Nuovo colore voti

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#762] [Idee e proposte]
Buonasera volevo richiedere di mettere i voti non in media in blu.
Sarebbe possibile già farlo?

**Comment by robertopadovesi** [comment]
Solitamente i voti blu, rappresentano i voti inseriti da un altro docente.

**Reply by federicabianchi08** [reply]
Esatto, sono problemi piuttosto banali. Alla fine, non fa molta differenza se un voto che non fa media è blu o grigio. Dettagli del genere non dovrebbero complicare l'uso del registro.
